packages/volto/src/components/manage/Blocks/Video/Edit.jsx

```diff
@@ -22,6 +23,11 @@ const messages = defineMessages({
     id: 'Type a Video (YouTube, Vimeo or mp4) URL',
     defaultMessage: 'Type a Video (YouTube, Vimeo or mp4) URL',
   },
+  allowedURLs: {
+    id: 'YouTube, Vimeo, Peertube ({instances}) instance or mp4 URL allowed',
+    defaultMessage:
+      'YouTube, Vimeo, Peertube ({instances}) instance or mp4 URL allowed',
```

Comment:
Won't this be quite confusing if there are not actually any Peertube instances enabled in the configuration?

Reply:
Yes, it would be... any suggestions? :innocent:

Reply:
option 1: use 2 different messages depending on whether there are peertube instances or not

option 2: Use "Allowed sources: {sources}" as the message and format it with sources as a comma-separated list of sources.

(I think I like option 2 better)

Reply:
I've modified it